Update accounting-myob.md (#252)
codatio/codat-docs: docs/integrations/accounting/myob/accounting-myob.md

:::info MYOB AccountRight or Essentials?

> MYOB have migrated their Essentials API to a new version. The new version will be using the AccountRight API. You can read more about the update on <a className="external" href="https://developer.myob.com/api/myob-business-api/api-overview/getting-started/" target="_blank">MYOB website</a>.

Reviewer: ```suggestion
MYOB have migrated their Essentials API to a new version. The new version will be using the AccountRight API. You can read more about the update in the [MYOB developer centre](https://developer.myob.com/api/myob-business-api/api-overview/getting-started/).
```
